### ISSUE TITLE
Long polling /recieve post request

### ISSUE THREAD

**Issue by pkyad** [crossbario/crossbar#1775]
I have an old server : ubuntu 14.04 (python 2.7) crossbar (18.4.1) where i configured the long poll 

Perfect setup / working after doing some code changes ( implimented render_OPTIONS ) at several places in longpoll.py router to fix the CORS error

```
class WampLongPollResourceOpen(Resource):

  def render_OPTIONS(self, request):
     request.setHeader(b'Access-Control-Allow-Headers', b"Content-Type, Access-Control-Allow-Headers, Authorization, X-Requested-With")
     request.setHeader(b'access-control-allow-origin', b"*")
     return b""
```

-----------------------------------------------
On the new server Ubuntu 18.04 (crossbar 20.4.2)

with the same changes and most of the things are still fine but there is some issue with polling

/ws/recieve/

/ws/open/ , /ws/send/, works fine

But since /recieve/ is not working the entire connection resets itself 

Nothing urgent for us but really wanted to move the services to new one. Please help

**Comment by oberstet** [comment]
is there a problem with long-poll on the latest, unmodified crossbar release? if so, best would be to come up with a test case that reproduces this.

> we don't support old crossbar releases or custom modified versions

**Comment by pkyad** [comment]
Yes, the unmodified version (latest version) gives the CORS error over SSL connection but even if we fix the CORS by implimenting render_OPTIONS function the /recieve/ url is the one which never responds. 

other urls /send/ , /open/ etc does get the response.
